frontend/src/components/ShootEditor.vue

```diff
@@ -348,10 +405,14 @@ export default {
       this.snackbar = true
     }
   },
-  mounted () {
+  async mounted () {
     this.createInstance(this.$refs.container)
     this.update(this.value)
     this.refresh()
+
+    let shootCompletions = await getShootSpec()
+    shootCompletions = get(shootCompletions, 'data.spec.properties', {})
```

Comment:
better name than shootCompletions? This is the openApi spec for a shoot, right?

Reply:
renamed